File: Assingment 1/client/js/models/game.js
window.game = function(game) {
    this.board = [];
    this.influenceCards = game._influenceCards;
    this.inProgress = game._inProgress;
    this.gameName = game.name;
    this.numPlayers = game._userPlayers.length;

    for (let i = 0; i < game._board.length; i++) {
        const row = [];
        
        for (let j = 0; j < game._board[i].length; j++) {
            row.push(new window.block(game._board[i][j]));
        }

        this.board.push(row);
    }

    this.getInfluenceCards = function(playerId) {

    }
}

File: Assignment 1/clients/bot/src/main/java/Bot/models/Game.java
package Bot.models;
import java.util.*;

public class Game {
    public int[][] board;
    public Map<Integer, Set<InfluenceCard>> influenceCards;
    public ArrayList<UserPlayer> players;
    public Boolean inProgress;
    public Boolean finished;
    public String name;
    public int playerTurn;

    public int getNumPlayers() {
        return players.size();
    };

    public Set<InfluenceCard> getPlayerCards(int playerId) {
        return influenceCards.get(playerId);
    }

    public UserPlayer getPlayer(int playerId) {
        UserPlayer player = null;

        for (int i = 0; i < players.size(); i++) {
            UserPlayer temp = players.get(i);
            if (temp.playerId == playerId) {
                player = temp;
                break;
            }
        }

        return player;
    }
}